week1/citibike.sh

```diff
@@ -1,23 +1,25 @@
 #!/bin/bash
 #
 # add your solution after each of the 10 comments below
-#
+# reference https://www.unix.com/unix-for-dummies-questions-and-answers/133603-how-remove-string-specific-column-file.html
 
 # count the number of unique stations
-
+cut -d, -f4 201402-citibike-tripdata.csv | sort | uniq | wc -l
 # count the number of unique bikes
-
+cut -d, -f12 201402-citibike-tripdata.csv | sort | uniq | wc -l
 # count the number of trips per day
-
+cut -d, -f2 201402-citibike-tripdata.csv | cut -c1-11 | sort | uniq -c
 # find the day with the most rides
-
+cut -d, -f2 201402-citibike-tripdata.csv | cut -c1-11 | sort | uniq -c | sort -nr | head -n1 | cut -c3-7
```

Comment:
The last cut displays the number of rides on that day instead of the date, but everything else is right so this is fine.